Workbooks/AKS/Workload Details AKS/Workload Details AKS.workbook

```diff
@@ -5,7 +5,45 @@
       "type": 9,
       "content": {
         "version": "KqlParameterItem/1.0",
+        "crossComponentResources": [
+          "{Resource}"
+        ],
         "parameters": [
+          {
+            "id": "df1ba062-3122-406e-9679-602e1027d4c6",
+            "version": "KqlParameterItem/1.0",
+            "name": "Resource",
+            "type": 5,
+            "isRequired": true,
+            "value": "value::1",
+            "isHiddenWhenLocked": true,
+            "typeSettings": {
+              "resourceTypeFilter": {
+                "microsoft.containerservice/managedclusters": true,
```

Comment:
I see values in resourceTypeFilter not consistent.

Reply:
Sorry what is the specific issue of consistency here?

Reply:
> Sorry what is the specific issue of consistency here?
 Inconsistency i am referring here , other places values of resourceTypeFilter  has openshift related ones but not here. either we should remove  or keep the same list in all places.

Reply:
fixed